### ISSUE TITLE
lint jinja[spacing] warning when using backslash r ("\r")

### ISSUE THREAD

**Issue by hyberdk** [ansible/ansible-lint#2871]
##### Summary

when using "\r" in a string, ansible-lint reports spacing issue.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

Ubuntu 20.04 running WSL

```console (paste below)
ansible-lint --version
ansible-lint 6.10.2 using ansible 2.13.6
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

Create a playbook that looks like this:
```
---
- name: Test
  hosts: localhost
  tasks:
    - name: Lint fails with spacing issue
      ansible.builtin.debug:
        msg: "\r"
    - name: Lint fails with spacing issue sample two
      ansible.builtin.debug:
        msg: "blah\r"
    - name: Lint okay
      ansible.builtin.debug:
        msg: "test message"
```

In vscode, it will highlight the jinja[spacing] lint warning

![image](https://user-images.githubusercontent.com/3530739/210984447-0115bcf5-f95e-44e9-b43d-185bbe5aa5ea.png)

```console (paste below)
ansible-lint --offline test_lint.yml 
WARNING: ansible-lint is no longer tested under Python 3.8 and will soon require 3.9. Do not report bugs for this version.
WARNING  Skipped installing collection dependencies due to running in offline mode.
WARNING  Overriding detected file kind 'yaml' with 'playbook' for given positional argument: test_lint.yml
WARNING  Listing 1 violation(s) that are fatal
jinja[spacing]: Jinja2 spacing could be improved:  -> 
 (warning)
test_lint.yml:5 Jinja2 template rewrite recommendation: `
`.


              Rule Violation Summary               
 count tag            profile rule associated tags 
     1 jinja[spacing] basic   formatting (warning) 

Passed with min profile: 0 failure(s), 1 warning(s) on 1 files.
```

##### Desired Behavior

there should be no lint warning.

##### Actual Behavior

there a lint warning, it should not happen.

**Comment by hyberdk** [comment]
found duplicate.. in #2671 closing this again, sorry about the duplicate.
